app/analysis/index/controller.js

```diff
@@ -274,64 +248,67 @@ const AnalysisController = Controller.extend(AnalysisQueryParams.Mixin, {
       billingTypes: this.store.findAll("billing-type"),
       costCenters: this.store.findAll("cost-center"),
     });
-  }),
+  }
 
-  data: task(function* () {
+  @enqueueTask
+  *data() {
     const params = underscoreQueryParams(
       serializeParachuteQueryParams(this.allQueryParams, AnalysisQueryParams)
     );
 
-    const data = yield this.store.query("report", {
-      page: {
-        number: this._lastPage + 1,
-        size: 20,
-      },
-      ...params,
-      include: "task,task.project,task.project.customer,user",
-    });
+    if (this._canLoadMore) {
+      const data = yield this.store.query("report", {
+        page: {
+          number: this._canLoadMore ? this._lastPage + 1 : this._lastPage,
```

Comment:
Ternary check here is unnecessary since your already in an `if(this.canLoadMore)`